src/logger.js

```diff
@@ -29,6 +29,16 @@ logger.tryStringify = (value) => {
   }
 };
 
+logger.tryParse = (value) => {
```

Comment:
Why do we define this on the logger and not as a static utility method?  Same with `tryStringify`?  Feels like doing it as a utility method would be more testable and type safe.

Reply:
ye that's prolly fair